packages/stable/src/api/geospatial/types.ts

```diff
@@ -0,0 +1,267 @@
+import { InternalId, Limit, ExternalId } from '@cognite/sdk-core';
+
+import { Geometry } from 'geojson';
+
+export * from 'geojson';
+
+export type GeometryType = 'WKT' | 'GEOJSON';
+
+type GeometryPropertyType =
+  | 'GEOMETRY'
+  | 'POINT'
+  | 'LINESTRING'
+  | 'POLYGON'
+  | 'MULTIPOINT'
+  | 'MULTILINESTRING'
+  | 'MULTIPOLYGON'
+  | 'GEOMETRYCOLLECTION'
+  | 'GEOMETRYZ'
+  | 'POINTZ'
+  | 'LINESTRINGZ'
+  | 'POLYGONZ'
+  | 'MULTIPOINTZ'
+  | 'MULTILINESTRINGZ'
+  | 'MULTIPOLYGONZ'
+  | 'GEOMETRYM'
+  | 'POINTM'
+  | 'LINESTRINGM'
+  | 'POLYGONM'
+  | 'MULTIPOINTM'
+  | 'MULTILINESTRINGM'
+  | 'MULTIPOLYGONM'
+  | 'GEOMETRYCOLLECTIONM'
+  | 'GEOMETRYZM'
+  | 'POINTZM'
+  | 'LINESTRINGZM'
+  | 'POLYGONZM'
+  | 'MULTIPOINTZM'
+  | 'MULTILINESTRINGZM'
+  | 'MULTIPOLYGONZM'
+  | 'GEOMETRYCOLLECTIONZM';
+
+export type PropertyType =
+  | 'STRING'
+  | 'LONG'
+  | 'DOUBLE'
+  | 'BOOLEAN'
+  | 'TIMESTAMP'
+  | GeometryPropertyType;
+
+export type Property =
+  | {
+      type: GeometryPropertyType;
+      srid?: number;
+      description?: string;
+      optional?: boolean;
+    }
+  | {
+      type: Exclude<PropertyType, GeometryPropertyType | 'STRING'>;
+      description?: string;
+      optional?: boolean;
+    }
+  | {
+      type: 'STRING';
+      size: number;
+      description?: string;
+      optional?: boolean;
+    };
+
+export interface Properties {
+  [property: string]: Property;
+}
+
+export interface Geospatial {
```

Comment:
what is this interface for?

Reply:
This is place holder for the Geospatial class as it has feature & featureType inside it.

Reply:
should then feature & featureType be specified in the interface? or they are not accessible from that interface?